### PR TITLE
[INLONG-8298][Dashboard] Stream supports data preview

### PR DESCRIPTION
**Pull request by bluewang**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes https://github.com/apache/inlong/issues/8298

### Verifying this change

image

### CODE REVIEW

File: inlong-dashboard/src/ui/locales/en.json [hunk 1]
```diff
@@ -625,6 +625,11 @@
   "pages.GroupDetail.Audit.Hour": "Hour",
   "pages.GroupDetail.Audit.Day": "Day",
   "pages.GroupDetail.Audit.Sink": "Sink",
+  "pages.GroupDetail.Stream.Preview": "Data Preview",
+  "pages.GroupDetail.Stream.NodeIp": "DataNode IP",
+  "pages.GroupDetail.Stream.Dt": "Time",
```
Comment (healchow) [inline inlong-dashboard/src/ui/locales/en.json:630]:
```suggestion
  "pages.GroupDetail.Stream.Dt": "Time(dt)",
```

File: inlong-dashboard/src/ui/locales/en.json [hunk 2]
```diff
@@ -625,6 +625,11 @@
...
+  "pages.GroupDetail.Stream.Content": "Content",
```
Comment (healchow) [inline inlong-dashboard/src/ui/locales/en.json:631]:
```suggestion
  "pages.GroupDetail.Stream.Content": "Data Content",
```

**Comment by leezng** [comment]
image
Is this a pagination interaction? You can use the unified pagination of the table.

File: inlong-dashboard/src/ui/pages/GroupDetail/DataStream/PreviewModal.tsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    {
+      title: 'ID',
+      dataIndex: 'id',
+    },
+    {
+      title: i18n.t('pages.GroupDetail.Stream.NodeIp'),
+      dataIndex: 'nodeIp',
+    },
+    {
+      title: i18n.t('pages.GroupDetail.Stream.Dt'),
```
Comment (leezng) [inline inlong-dashboard/src/ui/pages/GroupDetail/DataStream/PreviewModal.tsx:57]: Time display needs to be formatted.

**Comment by bluewang** [comment]
> image Is this a pagination interaction? You can use the unified pagination of the table.

https://github.com/apache/inlong/pull/8299#issuecomment-1605822196

**Comment by bluewang** [comment]
> image Is this a pagination interaction? You can use the unified pagination of the table.

This is the number of data rows queried

**Comment by dockerzhang** [comment]
Number -> Records Number

File: inlong-dashboard/src/ui/pages/GroupDetail/DataStream/PreviewModal.tsx [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+  return (
+    <Modal
+      {...modalProps}
+      title={i18n.t('pages.GroupDetail.Stream.Preview')}
+      width={1000}
+      footer={null}
+    >
+      <div style={{ marginBottom: 20, marginTop: 20 }}>
+        <span>{i18n.t('pages.GroupDetail.Stream.Number')}: </span>
+        <Button
```
Comment (leezng) [inline inlong-dashboard/src/ui/pages/GroupDetail/DataStream/PreviewModal.tsx:97]: It is recommended to use [RadioButton](https://ant.design/~demos/components-radio-demo-radiobutton).
image

**Review by leezng — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**